### PR TITLE
chore: remove redirect of nvm install stderr in e2e test 

### PR DESCRIPTION
**Pull request by nolanmar511**

### CODE REVIEW

File: testing/integration_test.go [hunk 1]
```diff
@@ -76,7 +76,7 @@ export NVM_DIR="$HOME/.nvm" >/dev/null
 
 # nvm install writes to stderr and stdout on successful install, so both are
```
Comment (aalexand) [inline testing/integration_test.go]: Update the comment?

Comment (nolanmar511) [reply testing/integration_test.go]: Updated the comment, and based on the comment, decided to keep the redirect of stdout.

**Comment by nolanmar511** [comment]
PTAL

File: testing/integration_test.go (hunk 1 above)
Comment (aalexand) [reply testing/integration_test.go]: The comment still says "both are re-directed"? Didn't push the code?

Comment (nolanmar511) [reply testing/integration_test.go]: Yep -- forgot to push. Done.

File: testing/integration_test.go [hunk 2]
```diff
@@ -74,9 +74,10 @@ retry curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.8/install.
 export NVM_DIR="$HOME/.nvm" >/dev/null
 [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh" >/dev/null
 
-# nvm install writes to stderr and stdout on successful install, so both are
+# nvm install writes to stderr and stdout on successful install, so some
+# output will appear on successful installs. Only output to stdout is
```
Comment (aalexand) [inline testing/integration_test.go]: "Only output to stdout is redirected" - this documents "what", not "why". Please add a note on "why".

**Comment by aalexand** [comment]
@nolanmar511 I left one note, please address and merge.

File: testing/integration_test.go (hunk 2 above)
Comment (nolanmar511) [reply testing/integration_test.go]: Done.

**Pull request merged**